easybuild/easyconfigs/n/ncurses/ncurses-5.9-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,18 @@
+name = 'ncurses'
+version = '5.9'
```

Comment:
@migueldiascosta please remove this easyconfig? it serves no purpose, since it's not required to resolve the dependencies here

It's better not to have easyconfigs for ncurses 5.9 around for recent toolchains...